packages/flutter_tools/test/integration.shard/flutter_build_wasm_test.dart

```diff
@@ -47,11 +47,11 @@ void main() {
       'web_wasm'
     ));
     for (final String filename in const <String>[
-      'dart2wasm_runtime.mjs',
       'flutter.js',
       'flutter_service_worker.js',
       'index.html',
       'main.dart.wasm',
+      'main.dart.mjs',
       'main.dart.js',
```

Comment:
Having both `main.dart.js` and `main.dart.mjs` is not ideal. I can rename `main.dart.js` to something, for example `wasm_bootstrap.js` seems like a reasonable name, but will that cause any issues with tooling? Alternatively, we could rename `main.dart.mjs` if you prefer.

Reply:
So yeah, as we discussed this is totally fine for now, and we can revisit it in the future. `main.dart.js` will be gone eventually and embedded directly into `flutter.js`, and it would be nice to have a command line arg to name `main.dart.mjs` whatever we want, and call it something a little more descriptive. But this is perfectly fine for the moment.